### DISCUSSION TITLE
clean-css 5

### DISCUSSION THREAD

**Discussion by jakubpawlowicz** [jakubpawlowicz/clean-css#1135] [Show and tell]
Hey everyone,

I've been preparing clean-css version 5 release for some time now. There are still many outstanding bugs I would like to get fixed in the upcoming release but I plan to squash many of them in the next few days.

The plan is to have the final version ready next week, but we'll see how it goes.
In the meantime please give the main branch a try, so we can identify any issues even before the release.

Thanks!
Jakub

**Comment by jakubpawlowicz** [comment]
Short update here: unless something unexpected happens, I plan to release version 5.0 tomorrow, Friday Jan 29th.

**Reply by leeoniya** [reply]
![image](https://user-images.githubusercontent.com/43234/106147348-5dd3e800-613d-11eb-8e44-b09841d74c7e.png)

**Comment by jakubpawlowicz** [comment]
Ok folks, clean-css 5.0 is finally out with many bugfixes and couple new features, see changelog: https://github.com/jakubpawlowicz/clean-css/blob/master/History.md#500--2021-01-29

I would like to thank you all for patience, this was long in making, but we are there now.
I have some plans for clean-css going forward, but let's leave that for another post next week.

Cheers!

**Reply by chalkygames123** [reply]
Awesome work! Thanks a lot 👏
